pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -224,218 +214,89 @@ func (rq *ResourceQuotaController) syncResourceQuotaFromKey(key string) (err err
 	return rq.syncResourceQuota(quota)
 }
 
-// syncResourceQuota runs a complete sync of current status
-func (rq *ResourceQuotaController) syncResourceQuota(quota api.ResourceQuota) (err error) {
-
+// syncResourceQuota runs a complete sync of resource quota status across all known kinds
+func (rq *ResourceQuotaController) syncResourceQuota(resourceQuota api.ResourceQuota) (err error) {
```

Comment:
Sorry to bother you, i have a quetion why we need a complete sync? client-go/Informer seems only rely on Watch API and can ensure the correct data. :) I'm a newbee.